docs/kubernetes.md

```diff
@@ -1,8 +1,10 @@
 # Kubernetes Concepts for Gluster Storage
 
-[Persistent Volumes] (http://kubernetes.io/docs/user-guide/persistent-volumes/)
-[Persistent Volume Implementation] (https://github.com/kubernetes/kubernetes/blob/master/docs/proposals/volume-provisioning.md)
-[Dynamic Provisioning and Storage Classes in Kubernetes] (http://blog.kubernetes.io/2016/10/dynamic-provisioning-and-storage-in-kubernetes.html)
+*[Persistent Volumes] (http://kubernetes.io/docs/user-guide/persistent-volumes/)
```

Comment:
The formatting is wrong: I think you need space between bullet point and text. See the result here:
https://github.com/ramkrsna/gluster-kubernetes/blob/addDocs/docs/kubernetes.md